src/impmal.js

```diff
@@ -74,6 +74,10 @@ import AugmeticItemSheet from "./sheet/items/item-augmetic-sheet.js";
 Hooks.once("init", () => 
 {
 
+    //shorten names
+    let Actors = foundry.documents.collections.Actors
+    let Items = foundry.documents.collections.Items
+
     // #if _ENV == "development"
```

Comment:
Let's instead convert this to just use `DocumentSheetConfig` for all sheet registration